readme.md

```diff
@@ -1,30 +1,63 @@
-# WIP: @jspm/node-importmap-loader
+# @jspm/node-importmap-loader
 
-##### Don't bother installing dependencies you don't need to! 🏇🏻💨
+![Typed with TypeScript](https://flat.badgen.net/badge/icon/Typed?icon=typescript&label&labelColor=blue&color=555555)
+[![npm version](https://badge.fury.io/js/node-importmap-http-loader.svg)](https://badge.fury.io/js/node-importmap-http-loader)
+![ci](https://github.com/yowainwright/node-importmap-http-loader/actions/workflows/ci.yml/badge.svg)
+[![Github](https://badgen.net/badge/icon/github?icon=github&label&color=black)](https://github.com/yowainwright/node-importmap-http-loader)
 
-With **JSPM _node-importmap-loader_**, installing dependencies is a thing of the past! You can reference and execute them directly! Maximize productivity ⚡️, decrease security risk 🚔, and probably do other awesome things too! Use **JSPM _node-importmap-loader_** today!
+#### Don't bother installing dependencies you don't need to! 🏇🏻💨
```

Comment:
Can we change the description a little bit, We can rather focus on the use-cases. Than telling the users want they can and can't. And it's personal opinion though.